DBB_DOCS.md

```diff
@@ -0,0 +1,180 @@
+   ```
+
+For other languages and more examples of Delta table operations, see [_](quick-start.md).
+
+### Configuration
+
+Here are the steps to configure <Delta> for S3.
+
+#. Include `hadoop-aws` JAR in the classpath.
+
+   <Delta> needs the `org.apache.hadoop.fs.s3a.S3AFileSystem` class from the `hadoop-aws` package, which implements Hadoop's `FileSystem` API for S3. Make sure the version of this package matches the Hadoop version with which Spark was built.
+
+#. Set up S3 credentials.
+
+We recommend using [IAM roles](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles.html) for authentication and authorization. But if you want to use keys, here is one way is to set up the [Hadoop configurations](https://spark.apache.org/docs/latest/configuration.html#custom-hadoophive-configuration) (in Scala):
+
+   ```scala
+   sc.hadoopConfiguration.set("fs.s3a.access.key", "<your-s3-access-key>")
+   sc.hadoopConfiguration.set("fs.s3a.secret.key", "<your-s3-secret-key>")
+   ```
+
+## Amazon S3 (with multi-cluster writes)
+
+.. note::
+This support is new and experimental.
+
+As mentioned in [Amazon S3](#amazon-s3), out-of-the-box <Delta> only supports concurrent writes to S3 from a single Spark driver. However, multi-cluster write support to S3 can be obtained by configuring <Delta> to use the right `LogStore` implementation.
+
+This implementation, with class name `DynamoDBLogStore`, uses [DynamoDB](https://aws.amazon.com/dynamodb/) to provide the mutual exclusion that S3 is inherently lacking.
+
+.. warning::
+This multi-cluster writing solution is only safe when all writers use this `LogStore` implementation. Concurrent writes to the same Delta table from multiple Spark drivers with some drivers using out-of-the-box <Delta> and others using this experimental `LogStore` can lead to data loss.
+
+### Requirements
+- All of the requirements listed in [Amazon S3](#amazon-s3)'s Requirements section
+- JAR of the [delta-storage-dynamodb artifact](https://todo)
+- JAR of the [AWS SDK for Java](https://mvnrepository.com/artifact/com.amazonaws/aws-java-sdk)
+
+### Configuration
+
+.. warning::
+Misconfiguring this `LogStore` implementation (e.g., specifying the wrong DynamoDB table) can lead to data loss.
+
+#. First, follow the configuration steps listed in [Amazon S3](#amazon-s3)'s Configuration section
+
+#. Configure LogStore implementation for the scheme `s3`.
+
+   ```ini
+   spark.delta.logstore.s3.impl=io.delta.storage.DynamoDBLogStore
+   ```
+
+#. Include the JARs for `delta-storage-dynamodb` and `aws-java-sdk` in the classpath.
+
+#. Configure the `LogStore` implementation in your Spark Session.
+
+Each Spark Session will need to be configured with information necessary to instantiate the DynamoDB client.
+
+A list of per-session configurations and their defaults is given below:
+
+   ```
+   - spark.delta.DynamoDBLogStore.tableName - table name (defaults to 'delta_log')
```

Comment:
We should clearly mention that these properties are hadoop properties (not spark ones). For example setting them via `pyspark --conf` will not work (additional `spark.hadoop.` prefix is required). I would suggest renaming these properties in DynamoDBLogStore driver to `delta.DynamoDBLogStore.*` and mention they can be set as other spark properties by adding `spark.hadoop.` prefix (or simply rename it here to `spark.hadoop.delta.DynamoDBLogStore.*`)

Reply:
@mrk-its good point. I'm actually working to solve this issue.

Seems pretty straight forward. In `LogStore.scala`, when we instantiate the java LogStore, we use `spark.sessionState.newHadoopConf` instead of `spark.sparkContext.hadoopConfiguration`.

This will copy over all the SQLConf values. Work in progress. https://github.com/delta-io/delta/issues/1022